### ISSUE TITLE
how to mapping the PCA shape components to Apple 51 blendshapes?

### ISSUE THREAD

**Issue by jinfagang** [LizhenWangT/FaceVerse#2]
Does there any instructions on how to convert?

**Comment by LizhenWangT** [comment]
I tried to build this conversion. But our model reconstructed from the East Asian faces is quite different with Apple's face model in the nose, eyes, etc. There is still some problems in this conversion. I will continue to try to solve this problem.

**Comment by jinfagang** [comment]
@LizhenWangT that is promising! Are u trying using Apple's standared face model to fit FaceVerse's version? It seems once we have a dividable 52 shapes, and it's semantic meaning, we can mimick apple's behavior.

**Comment by LizhenWangT** [comment]
Yes, that's just what I want to do. I'm trying to create the 52 FaceVerse shapes which is in the same expression with Apple's standard 52 blendshapes.

**Comment by jinfagang** [comment]
@LizhenWangT I am hoping for your impressive result!

**Comment by LizhenWangT** [comment]
@jinfagang This has been solved in version 2!

**Comment by ForrestPi** [comment]
@LizhenWangT  Thanks your work， could you provide the related code about  converting PCA shape components to Apple 51 blendshapes？

**Comment by LizhenWangT** [comment]
> @LizhenWangT Thanks your work， could you provide the related code about converting PCA shape components to Apple 51 blendshapes？

You can refer to this [issue](https://github.com/LizhenWangT/FaceVerse/issues/7). It's more like a manul job rahter than coding.
